### ISSUE TITLE
Correcting lyrics

### ISSUE THREAD

**Issue by Calorion** [ddddxxx/LyricsX#470]
Is is possible to correct lyrics within LyricsX and have that be reflected within the original database?

**Comment by RoyceThomas** [comment]
you can try contacting the lyrics provider (highlighted in pink)
<img width="749" alt="Screen Shot 2021-04-21 at 8 00 03 pm" src="https://user-images.githubusercontent.com/54102527/115535651-4ccde980-a2dc-11eb-8237-d2b55f5a324b.png">

**Comment by Calorion** [comment]
Well, consider this a feature request then. This would be an awesome ability.

**Comment by RoyceThomas** [comment]
true, but i just resort to manually creating the lyrics on websites such as [Akari's LRC Maker](https://lrc-maker.github.io/) (open-source online simple LRC maker) ([GitHub link](https://github.com/magic-akari/lrc-maker)) and 
[LRC Maker](https://lrcmaker.com/) (online enhanced LRC maker and player) (i mainly use Akari's)
When you create lyrics using these websites they only do line by line and trying to create enhanced lyrics looks really stupid on LyricsX, so I'm stuck with line by line. But since most of my songs are only line by line anyway i don't mind the extra effort
